manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml

```diff
@@ -0,0 +1,34 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+PackageIdentifier: Swift.Toolchain
+PackageVersion: 5.5
```

Comment:
```suggestion
PackageVersion: 0.0.0.0
```

Reply:
![image](https://user-images.githubusercontent.com/11600822/134746426-06667f64-09d1-4739-ab1e-1d5d8e1123ad.png)

Reply:
This was discussed when we first imported the package - the package currently always indicates `0.0.0.0` for the version, though the toolchain version is different.

Reply:
https://github.com/microsoft/winget-pkgs/pull/19313#issuecomment-870860207